docs/src/devdocs/interpolations.md

```diff
@@ -22,30 +61,136 @@ Ferrite.reference_shape_hessians_gradients_and_values!
 Ferrite.shape_value_type(ip::Interpolation, ::Type{T}) where T<:Number
 ```
 
-### Required methods to implement for all subtypes of `Interpolation` to define a new finite element
+## [How to implement a new interpolation, `QTI`](@id devdocs-howto_new-interpolation)
+As an example, we will implement a `Q`uadratic `T`riangle `I`nterpolation
+(`QTI`), corresponding to `Lagrange{RefTriangle, 2}()`. This is described
+in the devdocs as the API is not considered fully stable yet.
 
-Depending on the dimension of the reference element the following functions have to be implemented
+```@example InterpolationExample
+using Ferrite # hide
+using Test #hide
+struct QTI <: ScalarInterpolation{RefTriangle, 2}
```

Comment:
ScalarInterpolation is not mentioend before, so we might want to add a short comment on this one (and VectorInterpolation and VectorizedInterpolation to make sure users get what is going on here).

Reply:
I think we should document these [here](https://ferrite-fem.github.io/Ferrite.jl/stable/reference/interpolations/). Especially it looks like `ip^vdim` is only documented in the tutorials...

Reply:
#1149